### ISSUE TITLE
Allow api schemes property to be configured

### ISSUE THREAD

**Issue by wongelz** [swagger-api/swagger-play#118]
Perhaps swagger.api.schemes in application.conf

**Comment by dschiavu** [comment]
Agree, this property should be configurable. Currently, the generated `swagger.json` file is missing the `schemes` property so the Swagger UI defaults to the `https` schema when executing the APIs and the user is unable to switch between `http` and `https` at all.

**Comment by dschiavu** [comment]
A temporary workaround is to append `url=http://host/swagger.json` to the Swagger UI URL, it will force the UI to load the specified spec file and it will use the `http` instead of `https` scheme.
